bower.json

```diff
@@ -30,6 +30,7 @@
   "dependencies": {
     "angular": "1.2 - 1.5",
     "angular-ext-q": "1.x",
-    "angular-cache": "2.4.x"
+    "angular-cache": "2.4.x",
+    "lodash": "4.0.1"
```

Comment:
Yes it should allow a range so as not to cause bower conflicts in projects. Do we know if lodash is following semver? Can we trust using `4.x`?